external-crates/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -97,11 +99,164 @@ impl<'a, 'b> TypeView for TypeWithLoader<'a, 'b> {
     }
 }
 
+pub trait InterpreterInterface {
```

Comment:
we should think of a different name for this and nothing is coming to mind right now ;) 
let's think about it